mod/calculated_metrics/set/ltype_rtype/record/year.rb

```diff
@@ -1,44 +1,5 @@
-# This file is needed for handling virtual answers
-
 include_set Type::MetricAnswer
```

Comment:
I moved this code to type/metric_answer.  This pattern of including the type set inside the ltype_rtype set seems to be a robust workaround for now, so I used it in records, too.